CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 2.0.0

```

Comment:
Is this module compatible with all versions of BuildMaster? If not, should we list the known compatible versions?